go.mod

```diff
@@ -15,7 +15,7 @@ require (
 	github.com/google/uuid v1.3.0
 	github.com/kong/deck v1.12.2
 	github.com/kong/go-kong v0.29.0
-	github.com/kong/kubernetes-testing-framework v0.14.2
+	github.com/kong/kubernetes-testing-framework v0.15.1
```

Comment:
This is [already in `main`](https://github.com/Kong/kubernetes-ingress-controller/blob/c86e7be43e1dcfff1298501007caa8aeb44889a3/go.mod#L18)